server/chat-plugins/helptickets.ts

```diff
@@ -404,6 +413,47 @@ export class HelpTicket extends Rooms.RoomGame {
 	static modlog(entry: PartialModlogEntry) {
 		Rooms.Modlog.write('help-texttickets' as ModlogID, entry);
 	}
+	static logTextResult(ticket: TicketState & {text: [string, string], resolved: ResolvedTicketInfo}) {
+		const entry = {
+			text: ticket.text,
+			resolved: ticket.resolved,
+			meta: ticket.meta,
+			created: ticket.created,
+			userid: ticket.userid,
+			type: ticket.type,
+			claimed: ticket.claimed,
+		};
+		const date = Chat.toTimestamp(new Date()).split(' ')[0];
+		void FS(`logs/tickets/${date.slice(0, -3)}.jsonl`).append(JSON.stringify(entry) + '\n');
```

Comment:
I think these should probably be in directories by month.

Reply:
Why a directory? That's a lot of tiny files that are pretty hard to sort apart.

Reply:
I'm not opposed, it just seems more difficult.

Reply:
Ripgrep seems more efficient than trying to readdir potentially many files out of one dir at once.

Reply:
I think it's better than ripgrepping over a directory containing hundreds if not thousands of files. I'm not sure why you're concerned about "trying to readdir potentially many files out of one dir at once" when my suggestion _reduces_ the number of files in each directory.

Reply:
Discussed on PS and resolved. It's one file a month, whereas this comment was made thinking it was one a day.

Reply:
Apparently I misunderstood and this adds one file per month, which is fine.